integration_tests/steps/two_sided_pay_as_clear.py

```diff
@@ -26,15 +26,16 @@
 def test_traded_energy_rate(context):
     def has_one_of_clearing_rates(trade, market):
         return any(isclose((trade.offer.price / trade.offer.energy), clearing_rate)
-                   for clearing_rate in market.state.clearing[trade.time])
+                   for clearing_rate in list(market.state.clearing.values()))
```

Comment:
really minor, but `values()` already returns an iterable object, no need convert it into a list.